library/build.gradle.kts

```diff
@@ -8,6 +8,9 @@ plugins {
     id("signing")
 }
 
+group = "com.github.vishwaprojects"
```

Comment:
What are you doing?

Reply:
@Reedyuk i am completely new to github and contributing can you please add uuid in scan function ,immediately need for my project ,BTW i was trying to use mine forked library sorry for that

Reply:
No worries. I will help you, when I get a moment.
For yourself for now, you can publish using mavenlocal to use it in your project locally, you can do this without your namespace. Just bump the version.

A tip for rookies, your commit message should be descriptive, not random words, it should represent a short summary so that others understand the reason for your change.

Reply:
thank you @Reedyuk I tried using maven local as well as jetpack.io but was not successfully eagerly waiting for you to accept my pull request so that it will reflect changes in it and could able to use it